src/app/modals/market-listing-expiration/listing-expiration.component.ts

```diff
@@ -24,14 +25,20 @@ export class ListingExpirationComponent {
   expiration: number = 0;
 
   expiredList: Array<ListingExpiryIface> = [
-    { title: '1 day', value: 1, estimateFee: new Amount(0) },
-    { title: '2 days', value: 2, estimateFee: new Amount(0) },
-    // @TODO uncomment the commented code once smsg issue goes fixed.
-    // { title: '4 days', value: 4, estimateFee: new Amount(0) },
-    // { title: '1 week', value: 7, estimateFee: new Amount(0) },
-    // { title: '2 weeks', value: 14, estimateFee: new Amount(0) },
-    // { title: '3 weeks', value: 21, estimateFee: new Amount(0) },
-    // { title: '4 weeks', value: 28, estimateFee: new Amount(0) }
+    { title: '1 day', value: 0, estimateFee: new Amount(0), isDisabled: false },
```

Comment:
Might just need to change this back to having 'value: 1' (instead of 'value: 0') 
:smiling_imp:

Reply:
my bad, I'll change it quickly :)